### PR TITLE
Feature/ivangarcia register

### PR DESCRIPTION
**Pull request by ivanggv2**

### CODE REVIEW

File: src/schemas/validateRegisterSchema.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import * as yup from "yup";
+
+export const validateRegisterSchema = yup.object().shape({
+  email: yup
+    .string()
+    .required("Email required")
```
Comment (AdrianArcoGonzalez) [inline src/schemas/validateRegisterSchema.ts:6]: Will be better to use a friendly message to the user like email can't be empty

File: src/components/Register/Register.test.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      const inputRepeatEmail = screen.getByLabelText(inputRepeatEmailText);
+      const button = screen.getByRole("button");
+
+      await userEvent.type(inputUser, userWrite);
+      await userEvent.type(inputPassword, passwordWrite);
+      await userEvent.type(inputRepeatPassword, passwordWrite);
+      await userEvent.type(inputEmail, emailWrite);
+      await userEvent.type(inputRepeatEmail, emailWrite);
+
+      expect(button).toBeInTheDocument();
```
Comment (AdrianArcoGonzalez) [inline src/components/Register/Register.test.tsx:36]: We can look for a method to don't have to repeat expects and use a loop method for it

File: src/components/Register/Register.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { Formik } from "formik";
+import { ErrorsHandler } from "../../interfaces/interfaces";
+import { validateRegisterSchema } from "../../schemas/validateRegisterSchema";
+import { RegisterFormikForm } from "../RegisterFormikForm/RegisterFormikForm";
+import RegisterStyled from "./RegisterStyled";
+
+export const Register = (): JSX.Element => {
+  const initialValues = {
```
Comment (AdrianArcoGonzalez) [inline src/components/Register/Register.tsx:8]: Remeber to use typescript for this

**Review by AdrianArcoGonzalez — COMMENTED** [review]
take a look at the comments

File: src/components/RegisterFormikForm/RegisterFormikForm.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          name="confirmPassword"
+          placeholder="********"
+          type="password"
+        />
+
+        <CustomInputField
+          class="form-input"
+          label="Email"
+          name="email"
+          placeholder="John@mail.com"
```
[the remark this answers is not in the document]
Comment (ivanggv2) [reply src/components/RegisterFormikForm/RegisterFormikForm.tsx:42]: done

File: src/components/RegisterFormikForm/RegisterFormikForm.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+  return (
+    <RegisterFormikFormStyled>
+      <Form noValidate autoComplete="off" className="form-container">
+        <h1 className="form-title">Create your account</h1>
+        <CustomInputField
+          class="form-input"
+          label="Username"
+          name="username"
+          placeholder="John"
```
[the remark this answers is not in the document]
Comment (ivanggv2) [reply src/components/RegisterFormikForm/RegisterFormikForm.tsx:18]: done

File: src/schemas/validateRegisterSchema.ts (hunk 1 above)
Comment (ivanggv2) [reply src/schemas/validateRegisterSchema.ts:6]: done

File: src/components/Register/Register.tsx (hunk 3 above)
Comment (ivanggv2) [reply src/components/Register/Register.tsx:8]: done

File: src/components/Register/Register.test.tsx (hunk 2 above)
Comment (ivanggv2) [reply src/components/Register/Register.test.tsx:36]: There are 2 different calls to userEvent

**Pull request merged**